sand/extension_interface/search.py

```diff
@@ -1,21 +1,23 @@
 from abc import ABC, abstractmethod
+from typing import Dict
 
 
 class ISearch(ABC):
     """ Search Interface to support searches from multiple
         KG datastores.
     """
+
     @abstractmethod
-    def find_class_by_name(self):
+    def find_class_by_name(self, search_text: str) -> Dict:
         """Search Class using name"""
         pass
 
     @abstractmethod
-    def find_entity_by_name(self):
+    def find_entity_by_name(self, search_text: str) -> Dict:
```

Comment:
we should move this function to a separated interface (`IEntitySearch` and `IOntologySearch`) so for the scenario that users upload domain ontologies and still want to use entities from wikidata, we can reuse the wikidata search and only need to implement a new class for searching classes/properties.

Reply:
then `WikidataSearch` implements two interfaces `IEntitySearch` and `IOntologySearch`.

Reply:
Updated the ISearch to IEntitySearch and IOntologySearch interface implementations under the extension_interface.